src/app/interfaces/resilver-job.interface.ts

```diff
@@ -0,0 +1,27 @@
+export interface ResilverTime {
+  $date: number;
+}
+
+export interface ResilverScan {
+  bytes_issued: number;
+  bytes_processed: number;
+  bytes_to_process: number;
+  end_time: ResilverTime;
+  errors: number;
+  function: string;
+  pause: number | null;
+  percentage: number;
+  start_time: ResilverTime;
```

Comment:
Looks like `ResilverTime` is redundant and can be replaced with `ApiTimestamp`
https://github.com/truenas/webui/blob/647edd15eceb53a1cac5edaf3118610d15f0ad6d/src/app/interfaces/api-date.interface.ts#L1-L3

Reply:
Nice catch. Will do